sale_automatic_workflow/models/stock_picking.py

```diff
@@ -4,20 +4,33 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import api, fields, models
+from odoo.tools import float_compare
 
 
 class StockPicking(models.Model):
     _inherit = "stock.picking"
 
     workflow_process_id = fields.Many2one(
-        comodel_name='sale.workflow.process',
-        string='Sale Workflow Process'
```

Comment:
why change this ?

Reply:
Well I didn't, it was the previous commit. Don't know if I can take the liberty to change again this line, and I prefer having two lines for that too :)